documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -145,11 +145,12 @@ the data. The mappings are described in the following table.
 .2+| Encoded binary data
 | `xsd:hexBinary` | String
 | `xsd:base64Binary` | String
-.3+| Miscellaneous types
+.4+| Miscellaneous types
 | `xsd:anyURI` | String
 | `samm:curie` | String
 | `rdf:langString`
 | A JSON object with a structure as described in the rules above.
+| `samm-c:Either` | A JSON object with the left field and the value, which is one of the types proposed in `xref:characteristics.adoc#either-characteristic[Either]`.
```

Comment:
Unfortunately we can't put this here, since `samm-c:Either` is not a datatype, but a Characteristic. We might need a separate section for this, something like "Specific payload mappings for Characteristics", or maybe something that sounds better than this.